bedrock/base/templates/includes/protocol/footer/footer-refresh.html

```diff
@@ -108,7 +108,7 @@ <h2 class="moz24-footer-label" data-testid="footer-heading-developers">
           {% set moco_link = 'href="%s" data-link-type="footer" data-link-text="Mozilla Corporation"'|safe|format(url('mozorg.home')) %}
           {% set mofo_link = 'href="https://foundation.mozilla.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer" rel="external noopener" data-link-type="footer" data-link-text="Mozilla Foundation"'|safe %}
           {{ ftl('footer-refresh-visit-mozilla-corporations', moco_link=moco_link, mofo_link=mofo_link) }}<br>
-          {{ ftl('footer-refresh-portions-of-this-content', url=url('foundation.licensing.website-content'), current_year=current_year|string) }}
+          {{ ftl('footer-refresh-portions-of-this-content', href='href="%s"'|safe|format(url('foundation.licensing.website-content')), current_year=current_year|string) }}
```

Comment:
@alexgibson Is this something you had in mind in https://github.com/mozilla/bedrock/pull/14967#discussion_r1738546250 …?

_(Feel free to pick it up if you'd prefer something else, I'm just trying to follow some other examples…)_

Reply:
This follows the convention of how other links are handled in the footer here, so i think this is fine yes. Thanks!

Reply:
OK, since the anchor tag format and the variable name are already published via ftl to pontoon, this is probably the only thing that can be done about it.